src/path-util.typ

```diff
@@ -123,37 +132,64 @@
 /// - segments (array): List of path segments
 /// -> float: Total length of the path
 #let length(segments) = {
-  return segments.map(segment-length).sum()
+  return segments.map(_segment-length).sum()
 }
 
 /// Get position on path
 ///
 /// - segments (array): List of path segments
-/// - t (float): Position (from 0 to 1)
-/// -> vector: Position on path
+/// - t (int,float,ratio): Absolute position on the path if given an
+///   float or integer, or relative position if given a ratio from 0% to 100%
+/// -> none,vector: Position on path as vector clamped to the paths begin/end
+///    position.
```

Comment:
Clamping note is in the wrong place, it should be on `point-on-segment`. `point-on-path` panics if it is out of range.

Reply:
Fixed. Forgot to add the CHANGELOG to the commit.